### PR TITLE
Add no deps param for install for manager-core

### PR DESCRIPTION
**Pull request by yoland68**

### CODE REVIEW

File: tests/glob_test/manager_core_test.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    captured_output = StringIO()
+    sys.stdout = captured_output
+
+    result, error_occurred = manager_core.call_cli_dependencies()
+
+    sys.stdout = sys.__stdout__
+
+    assert result is None
+    assert error_occurred is True
+    assert "[ComfyUI-Manager] Failed to execute the command 'python -m comfy_cli dependency'." in captured_output.getvalue()
```
Comment (mcmonkey4eva) [inline tests/glob_test/manager_core_test.py:37]: nl@eof in these files

File: glob/manager_server.py [hunk 2]
```diff
@@ -967,6 +972,7 @@ def sanitize(data):
 
 
 async def _confirm_try_install(sender, custom_node_url, msg):
+    import pdb; pdb.set_trace()
```
Comment (ltdrdata) [inline glob/manager_server.py:975]: What's this?

Comment (mcmonkey4eva) [reply glob/manager_server.py:975]: debugger lol

Comment (yoland68) [reply glob/manager_server.py:975]: lol, my bad, PR s def not ready

**Pull request closed without merging**